.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      type:
+        description: 'New version type [new-version | major | minor | patch]'
+        required: true
+        default: 'patch'
+jobs:
+  bumpVersion:
+    runs-on: ubuntu-24.04
+    permissions:
+      contents: write
+
+    steps:
+      - run: |
+          echo "New version type: ${{ inputs.type }}"
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build
+        run: npm run build
+      - name: Config git
```

Comment:
Couldn't we use the Bonitasoft actions that configured the git user with a bot and doing the checkout with a token belonging to the bot?

If I am the actor, I know that the email address will be wrong with this configuration